resources/nbd_loop_device/actions/run.yaml

```diff
@@ -0,0 +1,52 @@
+- hosts: [{{ host }}]
+  sudo: yes
+  tasks:
+    - name: install nbd-server
+      apt: name=nbd-server state=latest
+    - name: install nbd-client
+      apt: name=nbd-client state=latest
+    - name: prepare a file
+      command: truncate -s 10G {{path}} creates={{path}}
+    - name: check if loop for file is already created
+      shell: sudo losetup -a|grep {{path}}
+      register: loop_created
+      ignore_errors: True
+    - name: if loop is not created, create it
+      command: losetup -f {{path}}
+      when: loop_created|failed
+    - name: find loop device
+      shell: losetup -a|grep '{{path}}'|awk -F':' '{print $1}'
+      register: loop_device
+    - name: check if loop device has gpt
+      shell: "sudo parted {{ '{{' }} loop_device.stdout }} print | grep 'Partition Table: gpt'"
```

Comment:
do you need sudo here? You playbook is configured to run with sudo with:
`sudo: yes`